packages/xo-server/src/xapi/index.js

```diff
@@ -598,6 +610,13 @@ export default class Xapi extends XapiBase {
         tags,
         version: asInteger(version),
         xenstore_data,
+
+        // VM created Suspended
+        power_state: suspend_VDI !== undefined ? 'Suspended' : undefined,
+        suspend_VDI,
+        domain_type,
+        last_boot_CPU_flags,
+        last_booted_record,
```

Comment:
These are also passed when the VM is not suspended.

Can these fields cause issues?

Reply:
No, all this fields are only used in the XAPI when the `power_state` is `Suspended`.

Reply:
How will it behave if the VM is not suspended?

Reply:
They are not used by the creation method

Reply:
I Can give them to the method only if `suspend_VDI` is not `undefined`

Reply:
Maybe that would be best/safer.